app/src/docs/v1.api-spec.yaml

```diff
@@ -1738,6 +1743,61 @@ components:
       schema:
         type: boolean
         example: true
+    Query-Permissions:
+      in: query
+      name: permissions
+      description: >-
+        Boolean representing whether or not to include matching permissions
+      schema:
+        type: boolean
+        example: true
+    Query-Page:
+      in: query
+      name: page
+      description: >-
+        The index of the page to return. The index of first page is 1.
+        Must specify limit when defined.
+      schema:
+        type: number
+        example: 1
+    Query-Limit:
+      in: query
+      name: limit
+      description: >-
+        The page size, number of objects in a page.
+        Must specify page number when defined.
+      schema:
+        type: number
+        example: 10
+    Query-Sort:
+      in: query
+      name: sort
+      description: >-
+        Any possible object Column.
+      schema:
+        type: string
+        enum:
+          - id
+          - path
+          - public
+          - active
+          - createdBy
+          - updatedBy
+          - updatedAt
+          - bucketId
+          - name
+        example: name
+    Query-Order:
+      in: query
+      name: order
+      description: >-
+        Order in ascending or descending. Default to ascending order.
+      schema:
+        type: string
+        enum:
+          - asc
+          - desc
+        example: asc
```

Comment:
Style: These parameter component definitions should stay alphabetized across the entire parameter subsection as the rest of the parameters were previously alphabetical order already.